Define and activate release 16.2.1 for kvm (#1002)
giantswarm/releases: kvm/v16.2.1/README.md

# :zap: Giant Swarm Release v16.2.1 for KVM :zap:

> This release enables the cluster to define multiple registry mirrors.

Author: ```suggestion
This release enables the workload clusters to define multiple registry mirrors.
```

> This release updates several preinstalled apps running in the workload cluster to reduce Kubernetes API server load and etcd memory usage in certain installations.

Reviewer: I think this needs update

> This release updates several preinstalled apps running in the workload cluster to reduce Kubernetes API server load and etcd memory usage in certain installations.

Author: ```suggestion
This release enables the cluster to define multiple registry mirrors.
```

## Change details

> ### kvm-operator [3.18.6](https://github.com/giantswarm/app-operator/releases/tag/v3.18.6)

Author: ```suggestion
### kvm-operator [3.18.6](https://github.com/giantswarm/kvm-operator/releases/tag/v3.18.6)
```
